docs/setup/configuration.md

```diff
@@ -21,7 +21,7 @@ To control Mattermost channel subscriptions, use the command `/jira subscribe` i
 1. As a Jira System Administrator, go to **Jira Settings > System > WebHooks**.
```

Comment:
Would be helpful if step 2 mentioned they could run `/jira webhook jiraURL` to get setup instructions.